### DISCUSSION TITLE
How to define the rotation when trying insert textbox by page.insert_htmlbox()?

### DISCUSSION THREAD

**Discussion by stickyfinger7** [pymupdf/PyMuPDF#3720] [Q&A]
I'd like to insert some text to the page with rotation of certain angles (like 30 degree) when applying **insert_htmlbox()**,but how to achieve my purpose? Besides, although I know how to use argument **morph** in **insert_textbox()**, this method is not as convenient   as **insert_htmlbox()** in my project case.

Here's my code

```
html = f'''
        <div>
            {text}
        </div>
          '''
css=f'''
      * {{
          color: {color};
          font-size: {font_size*0.7}px;
      }}
      '''
story=Story(html=html,user_css=css)
rect_fit=story.fit_scale(rect).rect
page.insert_htmlbox(rect_fit, html,css=css,scale_low=0)
```

**Comment by joiemoie** [comment]
+1 on this question, I need the insert_htmlbox but it's missing the morph parameter which I need for the custom rotations.
